src/RE/P/PlayerCharacter.cpp

```diff
@@ -22,14 +22,21 @@ namespace RE
 		REL::Relocation<bool*> singleton{ RELOCATION_ID(517711, 404238) };
 		return *singleton;
 	}
-
+#ifndef SKYRIMVR
 	void PlayerCharacter::ActivatePickRef()
 	{
 		using func_t = decltype(&PlayerCharacter::ActivatePickRef);
 		REL::Relocation<func_t> func{ Offset::PlayerCharacter::ActivatePickRef };
 		return func(this);
 	}
-
+#else
+	void PlayerCharacter::ActivatePickRef(VR_DEVICE a_device)
```

Comment:
Could we treat this as having a default value? Concern is that code using the Flat version will outright fail in VR.

Reply:
Sure, set default value to `kHeadset`

Reply:
With the default, shouldn't flat have the parameter in the sig but the ifdef is to not pass it to `func(this)`? Actually, wouldn't it be ignored anyway?

Reply:
It would be ignored in flat yes. If we include VR_DEVICE in flat, it may cause some confusion.

In my opinion, I would prefer to have the dev explicitly choose the VR_DEVICE when adding VR support since `ActivatePickRef` can be called from many different contexts (eg "What I am looking at, activate that" vs "What I could activate, activate this")

Reply:
Ok I buy that. It's a bit redundant in terms of duplicated code but I guess I can live with it.